x/ethbridge/alias.go

```diff
@@ -30,7 +30,7 @@ var (
 	CreateOracleClaimFromEthClaim     = types.CreateOracleClaimFromEthClaim
 	CreateEthClaimFromOracleString    = types.CreateEthClaimFromOracleString
```

Comment:
This file can be removed entirely. It was used previously for an internal folder but that is now gone

Reply:
Yeah, will do this later, probably after clp is merged to avoid clashes with some app files removed there.